devdox/app/schemas/git_label.py

```
@@ -61,7 +62,7 @@ class GetGitLabelsRequest:
     def __init__(
         self,
         pagination: Annotated[RequiredPaginationParams, Depends()],
-        git_hosting: Optional[str] = Query(
+        git_hosting: Optional[GitHosting] = Query(
             None, description="Filter by git hosting service"
         ),
```

Ruff (0.12.2)

Do not perform function call `Query` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)